ios/FluentUI/Controls/SearchBar.swift

```diff
@@ -330,6 +348,16 @@ open class SearchBar: UIView {
         constraints.append(searchTextField.centerYAnchor.constraint(equalTo: searchTextFieldBackgroundView.centerYAnchor))
         constraints.append(searchTextField.heightAnchor.constraint(equalTo: searchTextFieldBackgroundView.heightAnchor, constant: -2 * Constants.searchTextFieldVerticalInset))
 
+        //progressSpinner
+        searchTextFieldBackgroundView.addSubview(progressSpinner)
+        progressSpinner.translatesAutoresizingMaskIntoConstraints = false
+
+        constraints.append(progressSpinner.leadingAnchor.constraint(equalTo: searchTextField.trailingAnchor, constant: Constants.clearButtonLeadingInset))
+        constraints.append(progressSpinner.widthAnchor.constraint(equalToConstant: Constants.clearButtonWidth))
```

Comment:
if there are leading and trailing anchors is there a need for width anchor?

Reply:
copied the constraints from below and missed evaluating them. Will take a look and remove redundant.

---
In reply to: [465867515](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r465867515) [](ancestors = 465867515)

Reply:
removed redundant  in iteration 2

---
In reply to: [465882946](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r465882946) [](ancestors = 465882946,465867515)